support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx

```diff
@@ -1,17 +1,18 @@
 /* eslint-disable react/no-unused-state */
 // @flow
 
-import React, { Component } from 'react';
+// $FlowIgnore - required for hooks
```

Comment:
I have not been able to convince flow that hooks are available in preact, hence this new `FlowIgnore` everywhere